canvas_modules/common-canvas/src/common-canvas/svg-canvas-renderer.js

```diff
@@ -1976,7 +1976,9 @@ export default class SVGCanvasRenderer {
 					return;
 				}
 				const nodeGrp = d3.select(d3Event.currentTarget);
-				this.raiseNodeToTop(nodeGrp);
+				if (!this.config.enableLinksOverNodes) {
```

Comment:
It's not a big thing but, can you move this to the list of 'if' conditions inside the raiseNodeToTop function to keep all the conditions together.

Reply:
Sure, I have updated accordingly.